src/main/java/io/supertokens/storage/mysql/config/MySQLConfig.java

```diff
@@ -250,6 +259,34 @@ public String getThirdPartyUsersTable() {
         return addPrefixToTableName(tableName);
     }
 
+    public String getPasswordlessUsersTable() {
+        String tableName = "passwordless_users";
+
+        if (passwordless_users_table != null)
+            return passwordless_users_table;
+
+        return addPrefixToTableName(tableName);
+
+    }
+
+    public String getPasswordlessDevicesTable() {
+        String tableName = "passwordless_devices";
+
+        if (passwordless_device_table != null)
+            return passwordless_device_table;
```

Comment:
No need to do this. Also, I dislike `if` statements without a bracket. Please be sure to read the contributing guide we have!

Reply:
noted.